File: crud_mysql/Delete.py
import mysql.connector

myConnection = mysql.connector.connect(

    host="localhost",
    user = "root",
    password = "",
    database = "crud"
)

myCursor = myConnection.cursor()

query = "DELETE FROM details WHERE age = 22 "

myCursor.execute(query)

myConnection.commit()

print(myCursor.rowcount, "record(s) deleted")

File: NodeJs_mysql_crud/Delete.js
var mysql = require('mysql');

var connection = mysql.createPool({
    host : 'localhost',
    user : 'root',
    password : '',
    database : 'crud'
});

var query_delete = " DELETE FROM details WHERE name = ? ";

connection.getConnection(function (err, connection) {
    connection.query(query_delete,['kamran'], function (err, res) {
        if(err) throw err;
        else {
            console.log('A record is removed !')
        }
    });
    connection.release();
});
